lib/Interpreter/BackendPasses.cpp

```diff
@@ -69,6 +70,52 @@ namespace {
 
 char KeepLocalGVPass::ID = 0;
 
+namespace {
+
+  // Add a suffix to the CUDA module ctor/dtor to generate an unique name.
+  // It's necessary for lazy compilation. Without suffix, cling can't
+  // differentiate the ctor/dtor of the first module from the ctor/dtor of the
+  // following modules.
+  class UnqiueCUDACtorDtorName : public ModulePass {
+    static char ID;
+
+    bool runOnGlobal(GlobalValue& GV, StringRef & ModuleName){
```

Comment:
`StringRef` shouldn't be passed by reference.

Reply:
also the parameters can probably be passed as `const`